users/templates/user_site/home.html

```diff
@@ -71,7 +93,7 @@ <h1 style="font-family: 'Bodoni Moda', serif; font-size: 4.8em;">{{ month_name }
                                                    {% if current_date.1 == day.1 and current_date.2 == day.2 %} border-style:solid; border-color: red; 
                                                    {% else %} border-right: 1px solid #C1C3D1; {% endif %}">
                         <div style="cursor: pointer;">
-                            <p class="tab" style="top: -{{ max_margin }}vh;
+                            <p class="tab"  onclick="day_view('{{ day.2 }}/{{ day.1 }}/{{ day.0 }}')" style="top: -{{ max_margin }}vh;
```

Comment:
In my opinion, the day-view should be displayed while the user clicks on the square of the chosen day, and not on the date.

Reply:
Hi @YaelD 😄 
I agree with your point of view.
But I have tried to add the option to click the whole square and it interacts "badly" with the clicking of each event.
In other words, when you click an event it will first open the day view and then redirect to the update event page,
so it looks a bit funny.
If someone has an idea on how to solve it I would like to hear it,
otherwise, if we want to implement your idea then we can remove the clicking of events on the main page, and just from the day view a user can be redirected to the update event page.

Tell me if this is what you want me to implement 😃 

https://user-images.githubusercontent.com/63960808/167944871-6df81e93-4683-48a6-a1de-0022e9b7a7a2.mp4

Reply:
Yes, in my opinion, it would be better to remove the clicking of events on the main page and allow the user to update an event only by clicking on the event in the day view.

Reply:
changed  as requested😄